universal-application-tool-0.0.1/app/forms/QuestionForm.java

```diff
@@ -3,16 +3,42 @@
 import com.google.common.collect.ImmutableMap;
 import java.util.Locale;
 import java.util.Optional;
+import services.question.QuestionDefinition;
 import services.question.QuestionDefinitionBuilder;
 import services.question.QuestionType;
+import services.question.TranslationNotFoundException;
 
 public class QuestionForm {
-  private String questionName;
-  private String questionDescription;
-  private String questionPath;
-  private String questionType;
-  private String questionText;
-  private String questionHelpText;
+  private String questionName = "";
+  private String questionDescription = "";
+  private String questionPath = "";
+  private String questionType = "";
+  private String questionText = "";
+  private String questionHelpText = "";
+
+  public QuestionForm() {}
+
+  public QuestionForm(Optional<QuestionDefinition> definition) {
```

Comment:
suggestion (non-blocking) this constructor should take a `QuestionDefinition` to keep the constructor more simple. and whatever is calling this can do something like

`QuestionForm form = qd.isPresent() ? new QuestionForm(qd) : new QuestionForm();`